_source/_posts/2021-01-24-osgeo-glossary-updated.adoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: custom-post
+title:  "First release of OSGeo Glossary (and how to cite)!"
+date:   2021-01-24 11:30:00 +0800
+categories: update
+---
+
+A new release to celebrate the new year:
+the https://wiki.osgeo.org/wiki/Lexicon_Committee[OSGeo Lexicon Committee]
+has released its first set of terms!
+
+The OSGeo Glossary is now filled with terms and ready to use.
+These terms have been consolidated from OSGeo projects and
+especially the OSGeo Geo4All newsletter.
+
+We thank the editing team for compiling, editing and refining
+the terms (in alphabetical order):
+
+* Felicity Brand, https://wiki.osgeo.org/wiki/Season_of_Docs_Application_2019[OSGeo Season of Docs 2019]
+* Naini Khajanchi, Okta
+* Prof. Nikos Lambrinos, Chief Editor of the https://www.osgeo.org/initiatives/geo-for-all/geo-newsletters-archive/[GeoForAll newsletter]
+* Reese Plews, Convenor of the https://github.com/ISO-TC211/TMG[ISO/TC 211 TMG (Terminology Management Group)]
+* Cameron Shorter, Chair of the Lexicon Committee
+* Ankita Tripathi, Tech Writer and Community Liaison
+* Ronald Tse, Ribose
+
```

Comment:
* Felicity Brand: Technical Writer and Member of [OSGeo Season of Docs 2019](https://wiki.osgeo.org/wiki/Season_of_Docs_Application_2019)
* Prof. Nikos Lambrinos: Chief Editor of the https://www.osgeo.org/initiatives/geo-for-all/geo-newsletters-archive/[GeoForAll newsletter]
* Reese Plews: Convenor of the https://github.com/ISO-TC211/TMG[ISO/TC 211 TMG (Terminology Management Group)]
* Codrina Marie Ilie: Coordinator of OSGeo initiatives, such as the international FOSS4G conference
* Rob Atkinson: Research Engineer; OGC; maintainer of OGC Definitions Server; standards and open source software developer
* Ronald Tse: Founder of Ribose, developer of Geolexica and Glossarist; official host of ISO/TC 211 MLGT
* Cameron Shorter: Chair of the Lexicon Committee
* Naini Khajanchi: Technical Writer
* Ankita Tripathi: Tech Writer, and Community Liaison

Special thanks to Prof. Erik Stubkjær for his assistance